Tests/UITest.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using TicTacToe;
+using Xunit;
+
+namespace Tests.TicTacToe {
+
+    public class UITest {
+
+        public UI _ui;
```

Comment:
Why is this public?

You use an underscore prefix here but not in production code.